src/Model/Photo.php

```diff
@@ -351,6 +351,178 @@ public static function delete(array $conditions, array $options = [])
 		return DBA::delete("photo", $conditions, $options);
 	}
 
+	/**
+	 * @brief This function is used by the fromgplus addon
```

Comment:
You know that the addon will be removed in that version since G+ isn't anymore?

Reply:
The addon has been removed, please check if this method is used anywhere else.